docusaurus/docs/reference/ui-extensions.md

```diff
@@ -123,10 +123,6 @@ The method returns the `AppPlugin` instance to allow for chaining.
 - [Update the path based on the context](../how-to-guides/ui-extensions/register-an-extension.md#update-the-path-based-on-the-context)
 - [Open a modal from the `onClick()`](../how-to-guides/ui-extensions/register-an-extension.md#open-a-modal-from-the-onclick)
 
-#### See also
-
-- [Best practices for adding links](../how-to-guides/ui-extensions/register-an-extension.md#best-practices-for-adding-links)
-
```

Comment:
Was this removed by accident?

Reply:
Yep this heading and section doesn't exist anymore (sorry should have mentioned that in the description)